pkg/migrate/SplitStatements_test.go

```diff
@@ -0,0 +1,47 @@
+package migrate
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSplitStatements(t *testing.T) {
+
+	// Load the fixture with the sql example
+	fixture := "./fixtures/copyFromStdin.sql"
+	f, err := os.Open(fixture)
+	require.Nil(t, err)
+
+	in := NewBuffer()
+	dropComments := true
+	dropBlankLines := true
+	dropSearchPath := false
+
+	ReadInSQL(f, in, dropComments, dropBlankLines, dropSearchPath)
+	formattedSQL := in.String()
+
+	lines := make(chan string, 1000)
+	//read buffer values into the channel
+	go func() {
+		scanner := bufio.NewScanner(strings.NewReader(formattedSQL))
```

Comment:
Can we just pass in the buffer?

```suggestion
		scanner := bufio.NewScanner(in)
```

Reply:
Aha, this is our buffer. So never mind. It's an instance of `strings.Builder` which doesn't implement `io.Reader`.